app/models/database/HeadersQueryRepository.scala

```diff
@@ -20,5 +20,8 @@ object HeadersQueryRepository {
 
   def findByHeightRangeQuery(from:Int, to: Int): ConnectionIO[List[Header]] =
     sql"SELECT * FROM headers WHERE height BETWEEN $from AND $to".query[Header].to[List]
+
+  def findByCountQuery(from:Int, count: Int): ConnectionIO[List[Header]] =
+    sql"SELECT * FROM headers WHERE height BETWEEN $from AND ($from + $count - 1)".query[Header].to[List]
```

Comment:
I think query should be smth like `SELECT * FROM headers WHERE height >=  $from LIMIT $count`